Cargo.toml

```diff
@@ -19,6 +19,11 @@ futures-rustls = { version = "0.26", default-features = false }
 async-ws = { version = "0.4.0", optional = true }
 webpki-roots = "0.26.0"
 
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
```

Comment:
let's make these optional and add a `json` feature flag, which is enabled by default, see chat message from Nov 10 for details